worlds/factorio/data/mod_template/control.lua

```diff
@@ -157,6 +157,7 @@ function on_player_created(event)
 {%- if silo == 2 %}
     check_spawn_silo(game.players[event.player_index].force)
 {%- endif %}
+    dumpInfo(player.force)
```

Comment:
i don't see this being used?

Reply:
Currently it is not being used, as `dumpInfo` in general is currently not used. Instead we are currently polling once a second regardless of need. However, as there seems to be a desire to eventually go back to the "only poll when necessary" system (see one of the TODOs in FactorioClient.py), I thought I'd make this feature compatible with that other system as well. (The `dumpInfo` in `on_player_created` specifically would then be required to update the `FactorioContext.multiplayer` flag when a second player joins.)